plugins/ulimit-adjuster/README.md

```diff
@@ -0,0 +1,36 @@
+## ulimit Adjuster Plugin
+
+This sample plugin can adjust ulimits for containers using pod annotations.
+
+### Annotations
+
+ulimits are annotated using the key
+`ulimits.nri.containerd.io/container.$CONTAINER_NAME`, which adjusts ulimits
+for `$CONTAINER_NAME`.  The ulimit names are the valid names of Linux resource
```

Comment:
are there scenarios where customer may want to set a single per-pod override that will apply to all containers?

Reply:
Possibly, but I don't have a specific use-case for that right now.  I was looking to just build the simplest thing for now; we can always add functionality later if it's desired.

Reply:
thank you! If there will be a scenario with some global setting for a Pod that will require all containers to adjust limits, it will make sense to have a single override. I am also not aware of scenarios that would need this.